roles/rabbitmq/templates/etc/serverspec/rabbitmq_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+require 'socket'
+
+files= ['shadow', 'passwd']
+files.each do |file|
+  describe file("/etc/#{file}") do
+    it { should contain 'rabbitmq'}
+  end
+end  
+
+describe command('cat /etc/nova/nova.conf|grep rabbit_password') do
+  its(:stdout) { should_not contain /(rabbit_password=\w{0,7})$/ }
+end
```

Comment:
cat and grep isn't the right pattern here. There are built in methods to check if certain things exist in a file. See other serverspec PRs for examples.